Source/Core/AudioCommon/WASAPIStream.h

```diff
@@ -8,6 +8,9 @@
 // clang-format off
 #include <Windows.h>
 #include <mmreg.h>
+#include <objbase.h>
```

Comment:
Actually they do. `wil/resource.h` **must** be included after objbase.h, so it may be better to keep it here. Or move it outside since alphabetical ordering happens to be correct?

Reply:
I would keep it inside this block to make it clear ordering is important. However, I've moved WRL include away to a separate block because that does not need to follow a specific ordering.

Reply:
If ordering does matter, leave it in there. I just want to make sure this isn't mixed when it could just be its own sorted block instead.